tests/Monolog/Handler/MongoDBHandlerTest.php

```diff
@@ -27,20 +29,20 @@ public function testConstructorShouldThrowExceptionForInvalidMongo()
 
     public function testHandleWithLibraryClient()
     {
-        if (!(class_exists('MongoDB\Client'))) {
+        if (!(class_exists(Client::class))) {
```

Comment:
Extra parenthesis not necessary.
```suggestion
        if (!class_exists(Client::class)) {
```